expandFns.php

```diff
@@ -419,3 +419,15 @@ function equivalent_parameters($par) {
     default: return array($par);
   }
 }
+
+function str_almost_the_same($str1, $str2) {  // For comparing strings with forgiveness
+  $str1 = mb_strtolower($str1); // Case-insensitive
```

Comment:
If you are doing the same thing to $str1 and $str2, could you create a separate function to treat them?  This avoids unnecessary code repetition. 
Then `str_almost_the_same` could be simply `return(strcasecmp(remove_irrelevant_bits($str1), remove_irrelevant_bits($str2)) == 0)`